packages/react/src/Hero/Hero.tsx

```diff
@@ -1,69 +1,94 @@
-import React, {forwardRef} from 'react'
+import React, {forwardRef, PropsWithChildren, useMemo} from 'react'
 import clsx from 'clsx'
 import styles from './Hero.module.css'
-import {Button} from '../Button'
-import {Heading} from '../Heading'
+import {Button, ButtonBaseProps} from '../Button'
+import {Heading, HeadingProps} from '../Heading'
 import {Text} from '../Text'
 
 import type {BaseProps} from '../component-helpers'
 import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/hero/base.css'
 
-type Action = {
-  text: string
+export type HeroProps = BaseProps<HTMLDivElement> & {
+  align?: 'start' | 'center'
+  children: React.ReactNode | React.ReactNode[] // Maybe use this from `FAQ`? type FAQRootProps = PropsWithChildren<BaseProps<HTMLElement>>
```

Comment:
I'd say the way to go here using ReactNode is correct, but cc @rezrah

Reply:
If you use PropsWithChildren, you shouldn't need to do this 🤔. Otherwise this is fine because it also accounts for multiple children without a fragment. Can we remove the comment after this is fixed too please.

Reply:
Swapped this out with `PropsWithChildren`!